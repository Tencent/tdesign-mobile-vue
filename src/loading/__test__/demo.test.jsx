/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import { mount } from '@vue/test-utils';
import baseVue from '@/loading/demos/base.vue';
import delayVue from '@/loading/demos/delay.vue';
import horzVue from '@/loading/demos/horz.vue';
import mobileVue from '@/loading/demos/mobile.vue';
import pureTextVue from '@/loading/demos/pure-text.vue';
import sizeVue from '@/loading/demos/size.vue';
import speedVue from '@/loading/demos/speed.vue';
import vertVue from '@/loading/demos/vert.vue';

const mapper = {
  baseVue,
  delayVue,
  horzVue,
  mobileVue,
  pureTextVue,
  sizeVue,
  speedVue,
  vertVue,
};

describe('Loading', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Loading ${demoName} demo works fine`, () => {
      const wrapper = mount(mapper[demoName]);
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
