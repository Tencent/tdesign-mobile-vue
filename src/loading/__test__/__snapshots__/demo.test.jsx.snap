// Vitest Snapshot v1

exports[`Loading > Loading baseVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-loading"
    data-v-4a68c35c=""
    style="font-size: 20px;"
  >
    <svg
      class="t-loading__gradient t-icon-loading"
      height="1em"
      style="animation-duration: 800ms; width: 20px; height: 20px;"
      version="1.1"
      viewBox="0 0 12 12"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <foreignobject
        height="12"
        width="12"
        x="0"
        y="0"
      >
        <div
          class="t-loading__gradient-conic"
        />
      </foreignobject>
    </svg>
    <!---->
    <!---->
  </div>
  <div
    class="t-loading"
    data-v-4a68c35c=""
    style="font-size: 20px;"
  >
    <span
      class="t-loading__spinner"
      style="animation-duration: 800ms; width: 20px; height: 20px;"
    >
      
      <i
        class="t-loading__spinner--line t-loading__spinner--line-1"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-2"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-3"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-4"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-5"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-6"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-7"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-8"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-9"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-10"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-11"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-12"
      />
      
    </span>
    <!---->
    <!---->
  </div>
  <div
    class="t-loading"
    data-v-4a68c35c=""
    style="font-size: 40px;"
  >
    <div
      class="t-loading__dots"
      style="animation-duration: 800ms; width: 40px; height: 40px;"
    >
      
      <div
        class="t-loading__dot"
        style="animation-duration: 0.8s; animation-delay: 0s;"
      />
      <div
        class="t-loading__dot"
        style="animation-duration: 0.8s; animation-delay: 0.26666666666666666s;"
      />
      <div
        class="t-loading__dot"
        style="animation-duration: 0.8s; animation-delay: 0.5333333333333333s;"
      />
      
    </div>
    <!---->
    <!---->
  </div>
  
</div>
`;

exports[`Loading > Loading delayVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="switch-wrap"
    data-v-559e6688=""
  >
    <div
      class="t-switch t-switch--medium"
      data-v-559e6688=""
    >
      <div
        class="t-switch__dot t-switch__dot--medium t-switch__dot--plain"
      >
        <div
          class="t-switch__label t-switch__label--medium"
        >
          <!---->
        </div>
      </div>
    </div>
    <div
      data-v-559e6688=""
    >
      请求结束，隐藏loading
    </div>
  </div>
  <div
    class="t-loading"
    data-v-559e6688=""
    style="font-size: 20px;"
  >
    <!---->
    <!---->
    <!---->
  </div>
  
</div>
`;

exports[`Loading > Loading horzVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="t-loading"
    data-v-c48bb9cc=""
    style="font-size: 20px;"
  >
    <svg
      class="t-loading__gradient t-icon-loading"
      height="1em"
      style="animation-duration: 800ms; width: 20px; height: 20px;"
      version="1.1"
      viewBox="0 0 12 12"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <foreignobject
        height="12"
        width="12"
        x="0"
        y="0"
      >
        <div
          class="t-loading__gradient-conic"
        />
      </foreignobject>
    </svg>
    <span
      class="t-loading__text"
    >
      加载中...
    </span>
    <!---->
  </div>
  <div
    class="t-loading"
    data-v-c48bb9cc=""
    style="font-size: 20px;"
  >
    <span
      class="t-loading__spinner"
      style="animation-duration: 800ms; width: 20px; height: 20px;"
    >
      
      <i
        class="t-loading__spinner--line t-loading__spinner--line-1"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-2"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-3"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-4"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-5"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-6"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-7"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-8"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-9"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-10"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-11"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-12"
      />
      
    </span>
    <span
      class="t-loading__text"
    >
      加载中...
    </span>
    <!---->
  </div>
  
</div>
`;

exports[`Loading > Loading mobileVue demo works fine 1`] = `
<div
  class="tdesign-mobile-demo"
  data-v-448d2bed=""
>
  <h1
    class="title"
    data-v-448d2bed=""
  >
    Loading 加载中
  </h1>
  <p
    class="summary"
    data-v-448d2bed=""
  >
    用于表示页面或操作的加载状态，给予用户反馈的同时减缓等待的焦虑感，由一个或一组反馈动效组成。
  </p>
  <div
    class="tdesign-mobile-demo-block"
    data-v-448d2bed=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        01 组件类型
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        纯图标
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      <div
        class="loading-demo--flex"
        data-v-448d2bed=""
      >
        
        <div
          class="t-loading"
          data-v-4a68c35c=""
          style="font-size: 20px;"
        >
          <svg
            class="t-loading__gradient t-icon-loading"
            height="1em"
            style="animation-duration: 800ms; width: 20px; height: 20px;"
            version="1.1"
            viewBox="0 0 12 12"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <foreignobject
              height="12"
              width="12"
              x="0"
              y="0"
            >
              <div
                class="t-loading__gradient-conic"
              />
            </foreignobject>
          </svg>
          <!---->
          <!---->
        </div>
        <div
          class="t-loading"
          data-v-4a68c35c=""
          style="font-size: 20px;"
        >
          <span
            class="t-loading__spinner"
            style="animation-duration: 800ms; width: 20px; height: 20px;"
          >
            
            <i
              class="t-loading__spinner--line t-loading__spinner--line-1"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-2"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-3"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-4"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-5"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-6"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-7"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-8"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-9"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-10"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-11"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-12"
            />
            
          </span>
          <!---->
          <!---->
        </div>
        <div
          class="t-loading"
          data-v-4a68c35c=""
          style="font-size: 40px;"
        >
          <div
            class="t-loading__dots"
            style="animation-duration: 800ms; width: 40px; height: 40px;"
          >
            
            <div
              class="t-loading__dot"
              style="animation-duration: 0.8s; animation-delay: 0s;"
            />
            <div
              class="t-loading__dot"
              style="animation-duration: 0.8s; animation-delay: 0.26666666666666666s;"
            />
            <div
              class="t-loading__dot"
              style="animation-duration: 0.8s; animation-delay: 0.5333333333333333s;"
            />
            
          </div>
          <!---->
          <!---->
        </div>
        
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-448d2bed=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        图标加文字横向
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      <div
        class="loading-demo--flex"
        data-v-448d2bed=""
      >
        
        <div
          class="t-loading"
          data-v-c48bb9cc=""
          style="font-size: 20px;"
        >
          <svg
            class="t-loading__gradient t-icon-loading"
            height="1em"
            style="animation-duration: 800ms; width: 20px; height: 20px;"
            version="1.1"
            viewBox="0 0 12 12"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <foreignobject
              height="12"
              width="12"
              x="0"
              y="0"
            >
              <div
                class="t-loading__gradient-conic"
              />
            </foreignobject>
          </svg>
          <span
            class="t-loading__text"
          >
            加载中...
          </span>
          <!---->
        </div>
        <div
          class="t-loading"
          data-v-c48bb9cc=""
          style="font-size: 20px;"
        >
          <span
            class="t-loading__spinner"
            style="animation-duration: 800ms; width: 20px; height: 20px;"
          >
            
            <i
              class="t-loading__spinner--line t-loading__spinner--line-1"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-2"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-3"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-4"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-5"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-6"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-7"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-8"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-9"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-10"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-11"
            />
            <i
              class="t-loading__spinner--line t-loading__spinner--line-12"
            />
            
          </span>
          <span
            class="t-loading__text"
          >
            加载中...
          </span>
          <!---->
        </div>
        
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-448d2bed=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        图标加文字竖向
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      <div
        class="loading-demo--flex"
        data-v-448d2bed=""
      >
        <div
          class="container"
          data-v-3ab7b9bc=""
          data-v-448d2bed=""
        >
          <div
            class="t-loading t-loading--vertical"
            data-v-3ab7b9bc=""
            style="font-size: 20px;"
          >
            <svg
              class="t-loading__gradient t-icon-loading"
              height="1em"
              style="animation-duration: 800ms; width: 20px; height: 20px;"
              version="1.1"
              viewBox="0 0 12 12"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <foreignobject
                height="12"
                width="12"
                x="0"
                y="0"
              >
                <div
                  class="t-loading__gradient-conic"
                />
              </foreignobject>
            </svg>
            <span
              class="t-loading__text"
            >
              加载中
            </span>
            <!---->
          </div>
          <div
            class="t-loading t-loading--vertical"
            data-v-3ab7b9bc=""
            style="font-size: 20px;"
          >
            <span
              class="t-loading__spinner"
              style="animation-duration: 800ms; width: 20px; height: 20px;"
            >
              
              <i
                class="t-loading__spinner--line t-loading__spinner--line-1"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-2"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-3"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-4"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-5"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-6"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-7"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-8"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-9"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-10"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-11"
              />
              <i
                class="t-loading__spinner--line t-loading__spinner--line-12"
              />
              
            </span>
            <span
              class="t-loading__text"
            >
              加载中...
            </span>
            <!---->
          </div>
        </div>
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-448d2bed=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        纯文字
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot with-padding"
    >
      
      <div
        class="loading-demo--flex"
        data-v-448d2bed=""
      >
        <div
          class="t-loading"
          data-v-448d2bed=""
          style="font-size: 20px;"
        >
          <!---->
          <span
            class="t-loading__text t-loading__text--only"
          >
            加载中...
          </span>
          <!---->
        </div>
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block"
    data-v-448d2bed=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        02 组件尺寸
      </h2>
      <!--v-if-->
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="loading-demo"
        data-v-38fef6e8=""
      >
        <div
          class="loading-demo__summary mb-16"
          data-v-38fef6e8=""
        >
          大尺寸
        </div>
        <div
          class="t-loading"
          data-v-38fef6e8=""
          style="font-size: 26px;"
        >
          <svg
            class="t-loading__gradient t-icon-loading"
            height="1em"
            style="animation-duration: 800ms; width: 26px; height: 26px;"
            version="1.1"
            viewBox="0 0 12 12"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <foreignobject
              height="12"
              width="12"
              x="0"
              y="0"
            >
              <div
                class="t-loading__gradient-conic"
              />
            </foreignobject>
          </svg>
          <span
            class="t-loading__text"
          >
            加载中...
          </span>
          <!---->
        </div>
      </div>
      <div
        class="loading-demo"
        data-v-38fef6e8=""
      >
        <div
          class="loading-demo__summary mt-24 mb-16"
          data-v-38fef6e8=""
        >
          中尺寸
        </div>
        <div
          class="t-loading"
          data-v-38fef6e8=""
          style="font-size: 22px;"
        >
          <svg
            class="t-loading__gradient t-icon-loading"
            height="1em"
            style="animation-duration: 800ms; width: 22px; height: 22px;"
            version="1.1"
            viewBox="0 0 12 12"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <foreignobject
              height="12"
              width="12"
              x="0"
              y="0"
            >
              <div
                class="t-loading__gradient-conic"
              />
            </foreignobject>
          </svg>
          <span
            class="t-loading__text"
          >
            加载中...
          </span>
          <!---->
        </div>
      </div>
      <div
        class="loading-demo"
        data-v-38fef6e8=""
      >
        <div
          class="loading-demo__summary mt-24 mb-16"
          data-v-38fef6e8=""
        >
          小尺寸
        </div>
        <div
          class="t-loading"
          data-v-38fef6e8=""
          style="font-size: 20px;"
        >
          <svg
            class="t-loading__gradient t-icon-loading"
            height="1em"
            style="animation-duration: 800ms; width: 20px; height: 20px;"
            version="1.1"
            viewBox="0 0 12 12"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <foreignobject
              height="12"
              width="12"
              x="0"
              y="0"
            >
              <div
                class="t-loading__gradient-conic"
              />
            </foreignobject>
          </svg>
          <span
            class="t-loading__text"
          >
            加载中...
          </span>
          <!---->
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block"
    data-v-448d2bed=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        03 加载速度
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        加载速度调整
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="loading-demo"
        data-v-0fadab4c=""
      >
        <div
          class="t-loading"
          data-v-0fadab4c=""
          style="font-size: 20px;"
        >
          <svg
            class="t-loading__gradient t-icon-loading"
            height="1em"
            style="animation-duration: 600ms; width: 20px; height: 20px;"
            version="1.1"
            viewBox="0 0 12 12"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <foreignobject
              height="12"
              width="12"
              x="0"
              y="0"
            >
              <div
                class="t-loading__gradient-conic"
              />
            </foreignobject>
          </svg>
          <span
            class="t-loading__text"
          >
            加载中...
          </span>
          <!---->
        </div>
      </div>
      <div
        class="slider-wrap"
        data-v-0fadab4c=""
      >
        <div
          class="t-slider"
          data-v-0fadab4c=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default t-slider__line--single"
              style=""
            >
              <div
                class="t-slider__dot"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      
      
    </div>
  </div>
</div>
`;

exports[`Loading > Loading pureTextVue demo works fine 1`] = `
<div
  class="t-loading"
  style="font-size: 20px;"
>
  <!---->
  <span
    class="t-loading__text t-loading__text--only"
  >
    加载中...
  </span>
  <!---->
</div>
`;

exports[`Loading > Loading sizeVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="loading-demo"
    data-v-38fef6e8=""
  >
    <div
      class="loading-demo__summary mb-16"
      data-v-38fef6e8=""
    >
      大尺寸
    </div>
    <div
      class="t-loading"
      data-v-38fef6e8=""
      style="font-size: 26px;"
    >
      <svg
        class="t-loading__gradient t-icon-loading"
        height="1em"
        style="animation-duration: 800ms; width: 26px; height: 26px;"
        version="1.1"
        viewBox="0 0 12 12"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <foreignobject
          height="12"
          width="12"
          x="0"
          y="0"
        >
          <div
            class="t-loading__gradient-conic"
          />
        </foreignobject>
      </svg>
      <span
        class="t-loading__text"
      >
        加载中...
      </span>
      <!---->
    </div>
  </div>
  <div
    class="loading-demo"
    data-v-38fef6e8=""
  >
    <div
      class="loading-demo__summary mt-24 mb-16"
      data-v-38fef6e8=""
    >
      中尺寸
    </div>
    <div
      class="t-loading"
      data-v-38fef6e8=""
      style="font-size: 22px;"
    >
      <svg
        class="t-loading__gradient t-icon-loading"
        height="1em"
        style="animation-duration: 800ms; width: 22px; height: 22px;"
        version="1.1"
        viewBox="0 0 12 12"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <foreignobject
          height="12"
          width="12"
          x="0"
          y="0"
        >
          <div
            class="t-loading__gradient-conic"
          />
        </foreignobject>
      </svg>
      <span
        class="t-loading__text"
      >
        加载中...
      </span>
      <!---->
    </div>
  </div>
  <div
    class="loading-demo"
    data-v-38fef6e8=""
  >
    <div
      class="loading-demo__summary mt-24 mb-16"
      data-v-38fef6e8=""
    >
      小尺寸
    </div>
    <div
      class="t-loading"
      data-v-38fef6e8=""
      style="font-size: 20px;"
    >
      <svg
        class="t-loading__gradient t-icon-loading"
        height="1em"
        style="animation-duration: 800ms; width: 20px; height: 20px;"
        version="1.1"
        viewBox="0 0 12 12"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <foreignobject
          height="12"
          width="12"
          x="0"
          y="0"
        >
          <div
            class="t-loading__gradient-conic"
          />
        </foreignobject>
      </svg>
      <span
        class="t-loading__text"
      >
        加载中...
      </span>
      <!---->
    </div>
  </div>
  
</div>
`;

exports[`Loading > Loading speedVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="loading-demo"
    data-v-0fadab4c=""
  >
    <div
      class="t-loading"
      data-v-0fadab4c=""
      style="font-size: 20px;"
    >
      <svg
        class="t-loading__gradient t-icon-loading"
        height="1em"
        style="animation-duration: 600ms; width: 20px; height: 20px;"
        version="1.1"
        viewBox="0 0 12 12"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <foreignobject
          height="12"
          width="12"
          x="0"
          y="0"
        >
          <div
            class="t-loading__gradient-conic"
          />
        </foreignobject>
      </svg>
      <span
        class="t-loading__text"
      >
        加载中...
      </span>
      <!---->
    </div>
  </div>
  <div
    class="slider-wrap"
    data-v-0fadab4c=""
  >
    <div
      class="t-slider"
      data-v-0fadab4c=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--default"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default t-slider__line--single"
          style=""
        >
          <div
            class="t-slider__dot"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  
</div>
`;

exports[`Loading > Loading vertVue demo works fine 1`] = `
<div
  class="container"
  data-v-3ab7b9bc=""
>
  <div
    class="t-loading t-loading--vertical"
    data-v-3ab7b9bc=""
    style="font-size: 20px;"
  >
    <svg
      class="t-loading__gradient t-icon-loading"
      height="1em"
      style="animation-duration: 800ms; width: 20px; height: 20px;"
      version="1.1"
      viewBox="0 0 12 12"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <foreignobject
        height="12"
        width="12"
        x="0"
        y="0"
      >
        <div
          class="t-loading__gradient-conic"
        />
      </foreignobject>
    </svg>
    <span
      class="t-loading__text"
    >
      加载中
    </span>
    <!---->
  </div>
  <div
    class="t-loading t-loading--vertical"
    data-v-3ab7b9bc=""
    style="font-size: 20px;"
  >
    <span
      class="t-loading__spinner"
      style="animation-duration: 800ms; width: 20px; height: 20px;"
    >
      
      <i
        class="t-loading__spinner--line t-loading__spinner--line-1"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-2"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-3"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-4"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-5"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-6"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-7"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-8"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-9"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-10"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-11"
      />
      <i
        class="t-loading__spinner--line t-loading__spinner--line-12"
      />
      
    </span>
    <span
      class="t-loading__text"
    >
      加载中...
    </span>
    <!---->
  </div>
</div>
`;
